circle.yml

```diff
@@ -66,10 +66,6 @@ deployment:
     owner: atlasdb
     commands:
       - ./scripts/circle-ci/publish-github-page.sh
-  publish-internal-docs:
-    branch: develop
-    owner: atlasdb
-    commands:
       - curl -s --fail $DOCS_URL | bash -s -- -r docs/requirements.txt $CIRCLE_BRANCH
```

Comment:
(discussed offline)

this curl on its own is hard to understand; let's add comments that this line publishes the internal docs and the script publishes external docs.